### PR TITLE
css to git

### PR DESCRIPTION
**Pull request by gorzat**
zrzut ekranu 2017-03-16 o 14 57 26

### CODE REVIEW

File: Project/style/_main-footer.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.main-footer {
+    background-color: lightblue;
+    font-size: 0.8rem;
+    height: 30px;
+    width: 100%;
+    position: fixed;
+    display: flexed;
```
Comment (matzimowski) [inline Project/style/_main-footer.scss]: flexed? R U sure? 🍰  + 🐔 = 🍔

File: Project/style/_content-header.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (vergilius) [inline Project/style/_content-header.scss]: unnecessary newline

File: Project/style/_main-nav.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    text-decoration: none;
+}
+
+.main-nav__element {
+    display: inline;
+    padding-left: 40px;
+}
+
+.main-nav__link {
+    color: ghostwhite;
```
Comment (matzimowski) [inline Project/style/_main-nav.scss:12]: 👻

**Review by matzimowski — APPROVED** [review]

**Review by vergilius — APPROVED** [review]

File: Project/style.css [hunk 4]
```diff
@@ -1,144 +1,113 @@
...
+  flex-direction: row; }
 
 .top-bar__left {
-    display: flex;
-    align-items: center;
-    height: 100%;
-}
+  display: flex;
+  align-items: center;
+  height: 100%; }
```
Comment (kdzwinel) [inline Project/style.css:21]: `}` in the same line ? I'm upset

File: Project/style/_main-footer.scss (hunk 1 above)
Comment (kdzwinel) [inline Project/style/_main-footer.scss]: ??

File: Project/style.css (hunk 4 above)
Comment (kdzwinel) [reply Project/style.css:21]: Ah, this is generated. Carry on

**Pull request merged**